matrix/src/jetty-split.windows.dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG jdk
+ARG sourceVersion
+
+# Unzip in a separate container so that zip file layer is not part of final image
+FROM mcr.microsoft.com/windows/servercore:1809 as builder
+ARG sourceVersion
+ADD https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-home/${sourceVersion}/jetty-home-${sourceVersion}.zip /server.zip
+RUN ["powershell", "-Command", "expand-archive -Path /server.zip -DestinationPath /server"]
+
+FROM winamd64/openjdk:${jdk}-jdk-windowsservercore-1809
```

Comment:
Is there a reason to use different images for this main container and for the `builder`? If we could use the same image it would make builds faster.

Reply:
I made sure that the former was the base image for the latter, so this is not an issue.